socket/src/main/java/net/runelite/client/plugins/socket/plugins/playerstatus/PlayerStatusPlugin.java

```diff
@@ -0,0 +1,746 @@
+/*
```

Comment:
this should be a completely different plugin, not bundled into the socket plugin.